_toc.yml

```diff
@@ -582,51 +582,61 @@ entries:
       - file: docs/products/clickhouse/getting-started
         title: Get started
-      - file: docs/products/clickhouse/sample-dataset
       - file: docs/products/clickhouse/concepts
         title: Concepts
         entries:
-          - file: docs/products/clickhouse/concepts/features-overview
-            title: Features overview
-          - file: docs/products/clickhouse/concepts/service-architecture
-            title: Service architecture
           - file: docs/products/clickhouse/concepts/olap
           - file: docs/products/clickhouse/concepts/databases-and-tables
           - file: docs/products/clickhouse/concepts/columnar-databases
           - file: docs/products/clickhouse/concepts/indexing
       - file: docs/products/clickhouse/howto
         title: HowTo
         entries:
-          - file: docs/products/clickhouse/howto/connect-with-clickhouse-cli
-            title: Connect with ClickHouse client
-          - file: docs/products/clickhouse/howto/connect-with-go
-            title: Connect with Go
-          - file: docs/products/clickhouse/howto/connect-with-python
-            title: Connect with Python
-          - file: docs/products/clickhouse/howto/connect-with-nodejs
-            title: Connect with Node.js
-          - file: docs/products/clickhouse/howto/connect-with-php
-            title: Connect with PHP
-          - file: docs/products/clickhouse/howto/connect-with-java
-            title: Connect with Java
-          - file: docs/products/clickhouse/howto/secure-service
-            title: Secure a service
-          - file: docs/products/clickhouse/howto/manage-users-roles
-            title: Manage users and roles
-          - file: docs/products/clickhouse/howto/load-dataset
-            title: Load data
-          - file: docs/products/clickhouse/howto/query-databases
-            title: Query databases
-          - file: docs/products/clickhouse/howto/manage-databases-tables
-            title: Manage databases and tables
-          - file: docs/products/clickhouse/howto/monitor-performance
-            title: Monitor performance
-          - file: docs/products/clickhouse/howto/materialized-views
-            title: Use materialized views
-          - file: docs/products/clickhouse/howto/configure-access-terraform-deployed
-            title: Manage user privileges with Terraform
+          - file: docs/products/clickhouse/howto/list-get-started
+            entries:
+              - file: docs/products/clickhouse/howto/load-dataset
+                title: Load data
+              - file: docs/products/clickhouse/howto/secure-service
+                title: Secure a service
+          - file: docs/products/clickhouse/howto/list-connect-to-service
+            entries:
+              - file: docs/products/clickhouse/howto/connect-with-clickhouse-cli
+                title: Connect with ClickHouse client
+              - file: docs/products/clickhouse/howto/connect-with-go
```

Comment:
This is the same recommendation that was offered by @anelook on the previous pull request #1625 - consider grouping the code samples into their own directory, as we have done for other products. This is the example from PostgreSQL: https://docs.aiven.io/docs/products/postgresql/howto/list-code-samples.html